src/v2/components/ChannelPreview/hooks/useChannelPreviewDisplayName.ts

```diff
@@ -92,7 +92,14 @@ export const useChannelPreviewDisplayName = <
   const channelName = channel?.data?.name;
   const maxCharacterLength = characterLength || maxCharacterLengthDefault;
 
-  const [displayName, setDisplayName] = useState('');
+  const [displayName, setDisplayName] = useState(
+    getChannelPreviewDisplayName({
```

Comment:
Set initial value for displayName, to avoid blank view on load.

Reply:
the useEffect will technically trigger this to run on the first pass through. setting this state here as well will result in this occurring twice on the initial load which isn't ideal for performance

Reply:
Yeah thats what I thought. Can you see anything wrong here? https://github.com/GetStream/stream-chat-react-native/blob/sample-app-cleanup/examples/SampleApp/src/screens/ChannelScreen.tsx#L87

First render happens in empty string !!